docs/library/spi_engine/instruction-format.rst

```diff
@@ -265,29 +458,60 @@ bus behavior.
      - Configures the output of the three_wire pin.
    * - [1]
      - CPOL
-     - Configures the polarity of the SCLK signal. When 0, the idle state of
-       the SCLK signal is low. When 1, the idle state of the SCLK signal is
-       high.
+     - Configures the polarity of the SCLK signal:
+      
+       - When 0, the idle state of the SCLK signal is low.
+       - When 1, the idle state of the SCLK signal is high.
    * - [0]
      - CPHA
-     - Configures the phase of the SCLK signal. When 0, data is sampled on the
-       leading edge and updated on the trailing edge. When 1, data is
-       sampled on the trailing edge and updated on the leading edge.
+     - Configures the phase of the SCLK signal:
+      
+       - When 0, data is sampled on the leading edge and updated on the
+         trailing edge.
+       - When 1, data is sampled on the trailing edge and updated on the
+         leading edge.
+
+
+.. _spi_engine spi-lane-mask-register:
+
+SPI Lane Mask Register
```

Comment:
What is the plan to handle chips with a single SDO and multiple SDI lines? It seems like to be able to make those work in full-duplex, we would need a separate mask for each.